tracker/config.go

```diff
@@ -267,15 +296,15 @@ func newConfig(opts ...Options) *Config {
 				return c
 			}
 			if data["success"] == true {
-				account, ok := data["data"].(map[string]interface{})["account"].(string)
+				project_uid, ok := data["data"].(map[string]interface{})["project_uid"].(string)
 				if !ok {
 					log.Println("Failed to retrieve TenantID from  api response")
 					return c
 				}
 				if profilingServerUrl == "" {
-					profilingServerUrl = fmt.Sprint("https://" + account + ".middleware.io/profiling")
+					profilingServerUrl = fmt.Sprint("https://" + project_uid + ".middleware.io/profiling")
```

Comment:
We can consider using url.JoinPath() wherever we are doing concat with "+"

Reply:
@bhogayatakb Updated